examples/step-45/doc/results.dox

```diff
@@ -1,155 +1,10 @@
-                                               constraints);
-}
-@endcode
-
-The implementation of the recursive function should be mostly self explanatory
-given the discussion above. The
-<code>LaplaceProblem::make_periodicity_constraints()</code> function simply
-calls the former with matching faces of the first (and only) coarse mesh cell
-on refinement level 0. Note that when calling the recursive function we have
-to explicitly specify the template argument since the compiler can not deduce
-it (the template argument is only used in a "non-deducible context").
-
-This function is now dimension and finite element independent, but it still
-has the restriction that it assumes that the mesh is uniformly refined (or, in
-fact, only that matching periodic faces are refined equally). We check this at
-the beginning by asserting that both faces have the same number of children
-(that includes that neither have any children, i.e. that both are active).
-On the other hand, the function above can be extended to also allow this sort
-of thing. In that case, if we encounter the situation that only one cell is
-refined, we would have to recurse into its children and interpolate their
-degrees of freedom with respect to the degrees of freedom to the coarser
-matching face. This can use the same facilities the finite element classes
-already provide for computing constraints based on hanging nodes. We leave
-implementing this as an exercise, however.
-
-@note The functions above make one assumption, namely that the degrees of
-freedom on one face match one-to-one to the corresponding other face. This is
-sometimes difficult to establish, especially in 3d. For example, consider a
-long string of cube cells where we want to match the far left face with the
-far right face for periodicity. If the cells are all undistorted cubes, then
-everything will work as expected. But imagine we have twisted our cells so
-that the string as a whole now has a 90 degree twist; in that case, the
-coordinate systems of the far left and far right face are also rotated
-relative to each other, and the first DoF on the far left face will no longer
-be at the same location as the first DoF on the far right face. To make things
-a bit worse, the 3d case also allows for pathological cases where mesh cells
-are no longer orientable in the standard order (see the
-@ref GlossFaceOrientation "Face orientation" glossary entry), making matching
-coordinate systems difficult. Ultimately, whether you will encounter these
-cases depends on the kind of mesh you have: if your coarse mesh is just the
-unit cube or a subdivided hyper rectangle, you are definitely on the safe
-side. In other cases, if in doubt, verify the locations of degrees of freedom
-by printing the kind of information we have used in the tutorial's own
-implementation.
+<img src="http://www.dealii.org/images/steps/developer/step-45_non_periodic.png" alt="">
```

Comment:
There really is only a very small difference between these plots. Is the difference better visible if you plotted vectors instead of a scalar speed? Maybe the testcase is also not all that well chosen: your boundary conditions correspond to a purely rotational field, which means that it satisfies the Stokes equations, but is otherwise rather uninteresting. What if you chose something more interesting? E.g., through a localized right hand side vector, and zero boundary conditions at the inner and outer ring?